### PR TITLE
Dynamically generated html content

### PR DESCRIPTION
**Pull request by Bestbynature**
:zap: **Features included in this branch are:**
- When the user clicks (or taps) the button to check project details, the popup, with details about the project, appears. 💯 
- When the user clicks (or taps) the close (X) button, the popup disappears. 💯 
- JavaScript array to store all of the information for all projects. 💯 
- The main page is updated so that the projects section is created dynamically using the information stored in the JavaScript object. 💯

### CODE REVIEW

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi @Bestbynature and @melybee-07 👋 , 

Good job so far on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ No linter errors
✔️ Readme looks professional
✔️ PR summary present

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js [hunk 1]
```diff
@@ -42,4 +42,339 @@ links.forEach(link => {
...
+  back.textContent = 'Back End Dev';
+  line2.appendChild(back);
+
+  const dot2 = document.createElement('i');
+  dot2.className = 'fa-sharp fa-solid fa-circle span2';
+  line2.appendChild(dot2);
+
+  const year = document.createElement('h5');
+  year.className = 'span3';
+  year.textContent = portfolio[num].year;
```
[the remark this answers is not in the document]
Comment (Bestbynature) [reply index.js:252]: Thanks so much for this. It's resolved

[the remark this answers is not in the document]
Comment (Davidosky007) [reply index.js:254]: * Well done with all the efforts so far but your desktop design is still different from that of the figma 
*According to the figma design the popup is centered and focused all other items are blurred
## Your Design 
image
## Figma Design 
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @Bestbynature 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
* No linter errors 👍 
* Well written PR description 👍 
* Good looking readme file 👍 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidosky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by IngDavidTM — APPROVED** [review]
Hi @Bestbynature  🙋‍♂️, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**